src/GetSystemMetrics-Hook.cpp

```diff
@@ -38,28 +38,29 @@ VOID WINAPI MyExitProcess(UINT uExitCode)
 }
 
 // Install all detour hooks
-void InstallDetourHooks()
+static void InstallDetourHooks()
 {
     DetourTransactionBegin();
     DetourUpdateThread(GetCurrentThread());
 
-    // Hook GetSystemMetrics
-    if (DetourAttach(reinterpret_cast<PVOID *>(&Original_GetSystemMetrics), Hooked_GetSystemMetrics) != NO_ERROR)
+    struct HookPair
```

Comment:
I think that it would be better to declare the struct type at the top-level instead of inside of a function.